verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -441,17 +443,22 @@ export function SortSelect({
 }: SortSelectProps): JSX.Element {
     const classes = sortSelectStyles();
 
+    const filterBreadCrumbs = useAppSelector(selectFilterBreadcrumbs);
+    let filteredKeys = filterBreadCrumbs.map(({ key }) => key);  
+
     const sortKeywords = [
         { name: 'None', value: SortBy.Default },
         // Commenting out until fix is found for big queries sorting in mongodb
         // { name: 'Confirmed date', value: SortBy.ConfirmedDate },
-        { name: 'Country', value: SortBy.Country },
         { name: 'Location admin 1', value: SortBy.Admin1 },
         { name: 'Location admin 2', value: SortBy.Admin2 },
         { name: 'Location admin 3', value: SortBy.Admin3 },
         { name: 'Age', value: SortBy.Age },
     ];
 
+    !filteredKeys.includes('country') && sortKeywords.push({ name: 'Country', value: SortBy.Country });
```

Comment:
You can use `splice` method here to still keep `country` as the first filter in the dropdown. Like for example: 
`sortKeywords.splice(1, 0, { name: 'Country', value: SortBy.Country })`